<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHello_I_am_part_of_a_window">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuHello_I_am_part_of_a_window"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionAdventure_Record">
   <property name="text">
    <string>Adventure Record</string>
   </property>
  </action>
  <action name="actionArmour">
   <property name="text">
    <string>Armour</string>
   </property>
  </action>
  <action name="actionSkills">
   <property name="text">
    <string>Skills</string>
   </property>
  </action>
  <action name="actionTabed">
   <property name="text">
    <string>Tabed</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionNew</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>actionNew()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>actionSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>actionOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionArmour</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>actionArmour()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>603</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdventure_Record</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>actionAdventureRecord()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>603</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSkills</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>actionSkills()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>603</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTabed</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>actionTabed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>603</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>actionOpen()</slot>
  <slot>actionSave()</slot>
  <slot>actionNew()</slot>
  <slot>actionArmour()</slot>
  <slot>actionSkills()</slot>
  <slot>actionAdventureRecord()</slot>
  <slot>actionTabed()</slot>
 </slots>
</ui>
